src/wallet/wallet_rpc_server.cpp

```diff
@@ -4426,7 +4426,11 @@ namespace tools
       return false;
     }
 
-    if (!m_wallet->set_daemon(req.address, boost::none, req.trusted, std::move(ssl_options)))
+    boost::optional<epee::net_utils::http::login> daemon_login{};
+    if (!req.username.empty())
```

Comment:
`if (!req.username.empty() || !req.password.empty())` - I think the empty string for username is valid in the HTTP login spec.

Reply:
Added `|| !req.password.empty()` and left the minor version at 24. Please let me know if should be changed back to 23.

Reply:
Nope